modules/drivers/microphone/microphone_component.cc

```diff
@@ -72,30 +71,29 @@ bool MicrophoneComponent::Init() {
   return true;
 }
 
+void MicrophoneComponent::fill_channel_data(int chunk_i) {
+  // next index of channel data to be filled
+  int pos = chunk_i * chunk_ * sample_width_;
+  for (int buff_i = 0; buff_i < chunk_size_; pos += sample_width_){
```

Comment:
Run "apollo.sh lint" every time before submitting the code.

Reply:
Sorry, miss this. Will fix.